package-examples/_template/README.md

```diff
@@ -32,3 +32,14 @@ This step should describe the process and exact steps to render the declared val
 ### Apply the package
 
 This step should describe the process and exact commands needed to deploy the resources to a live cluster.
+
+
+{{% hide %}}
```

Comment:
Why do you need this? It is just a template.

Reply:
We need to first execute `source ./scripts/setupVerify.sh` to access all the functions. I placed this in README.md files of book and reference which executes first. I didn't find such a file in package-examples, but writing in template worked fine.